README.md

```diff
@@ -75,33 +96,33 @@ my_second_app
 my_other_app
 ```
 
-You can also delete a particular database:
+You can delete a particular database:
 
 ```sh-session
 $ fauna delete-database my_other_app
 deleting database 'my_other_app'
 database 'my_other_app' deleted
 ```
 
-You can also `create`, `list`, and `delete` _keys_.
+And you can create, list, and delete keys.
 
 This is how you create a key for the database `my_app`:
 
-```sh-session
+```sh
 $ fauna create-key my_app
 creating key for database 'my_app' with role 'admin'
 
 created key for database 'my_app' with role 'admin'.
-secret: ****************************************
+secret: fnAFPULk2WAAQY9t4x0tduzuz85gC-suDbTnl7um # this will be different
```

Comment:
I'd prefer keeping the original or have something like <your key here>

Reply:
I totally a few words. Oh so the behavior used to be that we'd hid the secret and you aligned it?

Reply:
He says yes.